lib/Tapable.js

```diff
@@ -53,33 +53,37 @@ Tapable.mixin = function mixinTapable(pt) {
 	copyProperties(Tapable.prototype, pt);
 };
 
-Tapable.prototype.applyPlugins = function applyPlugins(name) {
+Tapable.prototype.applyPlugins = function applyPlugins(name, param1, param2) {
 	if(!this._plugins[name]) return;
```

Comment:
FYI: Ran [this gist](https://gist.github.com/bmeurer/576c6d397377a43876353c67c2eb50b0) with latest Node 8.3.0 locally and I do indeed see results similar to what I observed with Canary:

![Node 8.3.0 results](https://user-images.githubusercontent.com/815988/29248050-5f900ca0-800e-11e7-8794-5765d99a7295.png)